routers/install/install.go

```diff
@@ -71,25 +71,16 @@ func Init(next http.Handler) http.Handler {
 			Render:  rnd,
 			Session: session.GetSession(req),
 			Data: map[string]interface{}{
+				"i18n":          locale,
 				"Title":         locale.Tr("install.install"),
 				"PageIsInstall": true,
 				"DbTypeNames":   getDbTypeNames(),
-				"i18n":          locale,
-				"Language":      locale.Language(),
-				"Lang":          locale.Language(),
 				"AllLangs":      translation.AllLangs(),
-				"CurrentURL":    setting.AppSubURL + req.URL.RequestURI(),
```

Comment:
Was this removal intended? There are still other places which set it and `head_navbar.tmpl` uses it.

Reply:
The install page do not need it, I have searched the code base.